### PR TITLE
Adding e2e for simpe send flow using default gas

### LINKED ISSUE #5695: [e2e] Send ETH from inside MetaMask using default gas

### PR DESCRIPTION
**Pull request by jpuri**
Fixes: #5695

Adding e2e for simple send flow using default gas.

The e2e succeeded only after I added some GoreliETH to the account. I am not sure what is right way to get an account with balance.

cc @seaona , @cortisiko

### CODE REVIEW

File: e2e/viewHelper.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    //
+  }
+};
+
+export const switchToGoreliNetwork = async () => {
+  await WalletView.tapNetworksButtonOnNavBar();
+  await NetworkListModal.changeNetwork(GOERLI);
+  await WalletView.isNetworkNameVisible(GOERLI);
+  await NetworkEducationModal.tapGotItButton();
+};
```
Comment (jpuri) [inline e2e/viewHelper.js:60]: These helpers will be useful as we plan to write more e2e for send flow.

File: e2e/specs/send-eth-flow.spec.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import TransactionConfirmationView from '../pages/TransactionConfirmView';
+import WalletView from '../pages/WalletView';
+import {
+  importWalletWithRecoveryPhrase,
+  switchToGoreliNetwork,
+} from '../viewHelper';
+
+const MYTH_ADDRESS = '0x1FDb169Ef12954F20A15852980e1F0C122BfC1D6';
+
+describe('Addressbook Tests', () => {
```
Comment (cortisiko) [inline e2e/specs/send-eth-flow.spec.js:15]: rename the test description

File: e2e/viewHelper.js (hunk 1 above)
Comment (cortisiko) [reply e2e/viewHelper.js:60]: Yup yup, I like this solution.

File: e2e/specs/send-eth-flow.spec.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
+'use strict';
```
Comment (cortisiko) [inline e2e/specs/send-eth-flow.spec.js:1]: can you create a subfolder called "Confirmations" in the specs folder, this way we put all the confirmations e2e tests

**Review by cortisiko — CHANGES_REQUESTED** [review]

File: e2e/specs/send-eth-flow.spec.js (hunk 3 above)
Comment (jpuri) [reply e2e/specs/send-eth-flow.spec.js]: Good Idea I implemented this.

**Comment by DigitalsupportTeam** [comment]
We'd be happy to look into this for you
kindly file a support ticket below 
https://forms.gle/Wmpm6NLzYm9uXYuU9

**Review by cortisiko — APPROVED** [review]
looks good to me. We are going to hide the SRP used in the tests behind an env variable. You can track the ticket [here](https://github.com/MetaMask/mobile-planning/issues/672).

**Pull request merged**